### PR TITLE
104-XXXX Errors

### PR DESCRIPTION
**Pull request by Spletz47**
### Changes:

Provides additional error code information for 104-XXXX error codes.

- [x] I have read and agreed to the [Code of Conduct](https://github.com/PretendoNetwork/Pretendo/blob/master/.github/CODE_OF_CONDUCT.md).
- [x] I have read and complied with the [contributing guidelines](https://github.com/PretendoNetwork/Pretendo/blob/master/.github/CONTRIBUTING.md).
- [ ] What I'm implementing was an [approved issue](../issues?q=is%3Aopen+is%3Aissue+label%3Aapproved).
- [x] I have tested all of my changes.

### CODE REVIEW

File: data/104/0440/en_US.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+"104": {
+    "0440": {
+        "name": "Unknown",
+        "message": "An error has occurred./n/nPlease try again later./n/nIf the problem persists, please\nmake a note of the error code and\nvisit support.nintendo.com.",
+        "short_description": "Unknown cause.",
+        "long_description": "Unknown cause.",
+        "short_solution": "Unknown solution.",
+        "long_solution": "Unknown solution.",
+        "support_link": "https://preten.do/104-0440"
+    }
```
Comment (jonbarrow) [inline data/104/0440/en_US.json:11]: We use tabs not spaces

File: data/104/0440/en_US.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+}
+}
```
Comment (jonbarrow) [inline data/104/0440/en_US.json:13]: Extra `}`

File: data/104/2210/en_US.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
 	"104": {
 		"2210": {
-			"name": "Unknown",
-			"message": "",
+			"name": "STOPPED_BY_POLICYLIST",
```
Comment (jonbarrow) [inline data/104/2210/en_US.json:4]: Where did this name come from?

File: data/104/2210/en_US.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+			"message": "An error has occurred./n/nPlease try again later./n/nIf the problem persists, please\nmake a note of the error code and\nvisit su",
```
Comment (jonbarrow) [inline data/104/2210/en_US.json:5]: This got cut off...?

File: data/104/2210/en_US.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 			"short_description": "Related to downloaded BOSS (SpotPass) content",
 			"long_description": "This error is related to your downloaded BOSS (SpotPass) data for the game you are playing. Either the content is missing or downloaded incorrectly. Most often seen in Splatoon.",
-			"short_solution": "Ensure Nimble is installed and set to auto-boot. You may also try deleting your games shared save data",
+			"short_solution": "Delete shared save data via Sytem Settings > Data Management. This will not delete any of your save data. Also make sure that your installation of Inkay is on the latest version. You can update Inkay via the GitHub page, Homebrew App Store, or the Aroma Updater.",
```
Comment (jonbarrow) [inline data/104/2210/en_US.json:8]: Short solutions should be as short as possible. This is getting pretty long

**Review by jonbarrow — CHANGES_REQUESTED** [review]
There are several pretty obvious formatting issues here. Was this file auto-generated or something? Please double check your changes before making PRs, especially since you checked off that you tested this (which is evidently not true, considering the malformed JSON here)

File: data/104/2210/en_US.json (hunk 3 above)
Comment (Spletz47) [reply data/104/2210/en_US.json:4]: Quarky when they were figuring out how to fix 104-2210. But I have been attempting to figure out how to convert results to error codes on undocumented modules, which I haven't made much progress on in recent weeks as I'm completely stuck

**Pull request merged**